pkgs/by-name/sw/swayfx/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  swayfx-unwrapped,
+  sway,
+  # Used by the NixOS module:
+  withBaseWrapper ? true,
+  extraSessionCommands ? "",
+  withGtkWrapper ? false,
+  extraOptions ? [], # E.g.: [ "--verbose" ]
+  isNixOS ? false,
+  enableXWayland ? true,
+  dbusSupport ? true,
+}:
+
+(sway.override {
+  inherit
+    withBaseWrapper
+    extraSessionCommands
+    withGtkWrapper
+    extraOptions
+    isNixOS
+    enableXWayland
+    dbusSupport
+    ;
+  sway-unwrapped = swayfx-unwrapped;
+}).overrideAttrs
+  {
+    name = "swayfx-${swayfx-unwrapped.version}";
```

```suggestion
    name = "swayfx-${swayfx-unwrapped.version}";
    inherit (swayfx-unwrapped) version;
```
or is this already set?

---

has `symlinkJoin` a version attribute?

---

The `sway` wrapper didn't have a version attribute, so I fixed it by setting the version to the `sway-unwrapped` version (which in `swayfx` gets overridden to `swayfx-unwrapped`).

---

> has `symlinkJoin` a version attribute?

I think we can just add it.